semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/AstNodeMethods.scala

```diff
@@ -55,4 +58,17 @@ class AstNodeMethods(val node: nodes.AstNode) extends AnyVal {
                        })
   }
 
+  def parentExpression: nodes.Expression = _parentExpression(node)
```

Comment:
There is an `astParent` method in AstNode.scala. But no corresponding version of it on AstNodeMethods.scala. I guess that is why you did not the the very similar naming.
Please add a comment what `parentExpression` does and how is compares to `astParent`.
Please also add a version of `parentExpression` to AstNode.scala in order to no let those to diverged even further.

Reply:
done in 8f5090e8ff5d4ea44a9db445c26bfd7135f043e0